site/docs/master/restic.md

```diff
@@ -160,6 +160,86 @@ You're now ready to use Velero with restic.
 
 ## Back up
 
+Velero supports two approaches of discovering pod volumes that need to be backed up using restic:
```

Comment:
`s/pod volumes/Pod Volumes`

Also: please capitalize resource names below.

Reply:
The existing docs in the repo all refer to this as "pod volumes".
Also this is not referring to a kubernetes type.